tcms/core/middleware.py

```diff
@@ -32,3 +34,20 @@ def process_request(self, request):
                       }
                 )
             )
+
+
+class CheckUnappliedMigrationsMiddleware(MiddlewareMixin):
+    def process_request(self, request):
+        executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
+        plan = executor.migration_plan(executor.loader.graph.leaf_nodes())
+        if plan:
+            apps_waiting_migration = sorted({migration.app_label for migration, backwards in plan})
+            messages.add_message(
+                request,
+                messages.ERROR,
+                mark_safe(
+                    _("\nYou have %(unapplied_migration_count)s unapplied migration(s). "
```

Comment:
Remove "\n" from the message.

Reply:
Cut the message at the first sentence and add a link to https://kiwitcms.readthedocs.io/en/latest/installing_docker.html#initial-configuration-of-running-container similar to the message about Base URL. We want to sent the user to the appropriate documentation.